queries/nu/highlights.scm

```diff
@@ -0,0 +1,396 @@
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+] @punctuation.bracket
+
+(val_record
+  (record_entry
+    ":" @punctuation.delimiter))
+
+key: (identifier) @property
+
+; ---
+; identifiers
+(param_rest
+  name: (_) @variable.parameter)
+
+(param_opt
+  name: (_) @variable.parameter)
+
+(parameter
+  param_name: (_) @variable.parameter)
+
+(param_cmd
+  (cmd_identifier) @string)
+
+(param_long_flag
+  (long_flag_identifier) @attribute)
+
+(param_short_flag
+  (param_short_flag_identifier) @attribute)
+
+(short_flag
+  (short_flag_identifier) @attribute)
+
+(long_flag_identifier) @attribute
+
+(scope_pattern
+  (wild_card) @function)
+
+(cmd_identifier) @function
+
+; generated with Nu 0.93.0
+; > help commands
+;   | filter { $in.command_type == builtin and $in.category != core }
+;   | each {$'"($in.name | split row " " | $in.0)"'}
+;   | uniq
+;   | str join ' '
+(command
+  head: [
+    (cmd_identifier) @function.builtin
+    (#any-of? @function.builtin
+      "all" "ansi" "any" "append" "ast" "bits" "bytes" "cal" "cd" "char" "clear" "collect" "columns"
+      "compact" "complete" "config" "cp" "date" "debug" "decode" "default" "detect" "dfr" "drop"
+      "du" "each" "encode" "enumerate" "every" "exec" "exit" "explain" "explore" "export-env" "fill"
+      "filter" "find" "first" "flatten" "fmt" "format" "from" "generate" "get" "glob" "grid" "group"
+      "group-by" "hash" "headers" "histogram" "history" "http" "input" "insert" "inspect"
+      "interleave" "into" "is-empty" "is-not-empty" "is-terminal" "items" "join" "keybindings"
+      "kill" "last" "length" "let-env" "lines" "load-env" "ls" "math" "merge" "metadata" "mkdir"
+      "mktemp" "move" "mv" "nu-check" "nu-highlight" "open" "panic" "par-each" "parse" "path"
+      "plugin" "port" "prepend" "print" "ps" "query" "random" "range" "reduce" "reject" "rename"
+      "reverse" "rm" "roll" "rotate" "run-external" "save" "schema" "select" "seq" "shuffle" "skip"
+      "sleep" "sort" "sort-by" "split" "split-by" "start" "stor" "str" "sys" "table" "take" "tee"
+      "term" "timeit" "to" "touch" "transpose" "tutor" "ulimit" "uname" "uniq" "uniq-by" "update"
+      "upsert" "url" "values" "view" "watch" "where" "which" "whoami" "window" "with-env" "wrap"
+      "zip")
+  ]
+  )
+
+(command
+  "^" @punctuation.delimiter
+  head: (_) @function)
+
+"where" @function.builtin
+
+(path
+  [
+    "."
+    "?"
+  ] @punctuation.delimiter) @variable.parameter
+
+(stmt_let
+  (identifier) @variable)
+
+(val_variable
+  "$" @punctuation.special
+  [
+    (identifier) @variable
+    "in" @variable.parameter.builtin
+    "nu" @module
+    "env" @constant
+  ]) @none
+
+(record_entry
+  ":" @punctuation.special)
+
+; ---
+; types
+(flat_type) @type
+
+(list_type
+  "list" @type.builtin
+  [
+    "<"
+    ">"
+  ] @punctuation.bracket)
+
+(collection_type
+  [
+    "record"
+    "table"
+  ] @type.builtin
+  "<" @punctuation.bracket
+  key: (_) @variable.parameter
+  [
+    ","
+    ":"
+  ] @punctuation.special
```

Comment:
Imo these should also be removed and be reaplced by a general capture (not the bracket ones though)